src/assets/scripts/client/InputController.js

```diff
@@ -722,51 +563,98 @@ export default class InputController {
     }
 
     /**
+     * Toggle command bar between aircraft commands and scope commands
+     *
      * @for InputController
-     * @method _parseUserCommand
-     * @return result {CommandParser}
+     * @method _toggleCommandBarContext
      */
-    _parseUserCommand() {
-        let result;
+    _toggleCommandBarContext() {
+        switch (this.commandBarContext) {
+            case COMMAND_CONTEXT.AIRCRAFT:
+                this.commandBarContext = COMMAND_CONTEXT.SCOPE;
+                this.$commandInput.attr('placeholder', 'enter scope command');
+                // this.$commandInput.css({'background-color': '#444'});
+                this.$commandInput.css({ color: 'red' });
+
+                return;
+
+            case COMMAND_CONTEXT.SCOPE:
+                this.commandBarContext = COMMAND_CONTEXT.AIRCRAFT;
+                this.$commandInput.attr('placeholder', 'enter aircraft command');
+                // this.$commandInput.css({ 'background-color': '#999' });
+                this.$commandInput.css({ color: 'white' });
+
+                return;
+
```

Comment:
please remove empty line between `return` and next `case`

Reply:
Done